workspaces/rbac/plugins/rbac/src/components/ConditionalAccess/ConditionalAccessSidebar.tsx

```diff
@@ -76,27 +41,56 @@ export const ConditionalAccessSidebar = ({
   conditionRulesData,
   conditionsFormVal,
 }: ConditionalAccessSidebarProps) => {
-  const classes = useDrawerStyles();
-  const contentClasses = useDrawerContentStyles();
   return (
     <Drawer
       anchor="right"
       open={open}
       data-testid="rules-sidebar"
-      classes={{
-        paper: classes.paper,
+      sx={{
+        ['@media (max-width: 960px)']: {
+          width: '-webkit-fill-available',
+        },
+        width: '50vw',
+        height: '100vh',
+        gap: '3%',
+        display: '-webkit-inline-box',
+        ['& div.MuiDrawer-paper']: {
+          width: '0px',
+        },
       }}
     >
-      <Box className={contentClasses.sidebar}>
-        <Box className={contentClasses.header}>
+      <Box
+        sx={{
+          display: 'flex',
+          flexFlow: 'column',
+          justifyContent: 'space-between',
+          height: '100%',
+          backgroundColor: theme =>
+            `${theme.palette.background.default} !important`,
```

Comment:
sx also support strings for picking up the theme. Idk what is "better". But please remove all `!important`, that shouldn't be necessary, or?